{"name": "test_update_user_with_incorrect_user_id_027", "status": "passed", "description": "\n    Отредактировать не существующего пользователя (не существующий user_id)\n\n    Ожидаемый результат:\n        Статус-код 404;\n        Время ответа сервера - не превышает 500ms;\n        Схема JSON-ответа соответствует Требованиям;\n        Response header \"Content-Type\": \"application/json\"\n        Response header \"Connection\": \"keep-alive\"\n        В JSON - присутствует ключ detail, значением является описание ошибки\n        В тексте ошибки указан отправленный нами \"user_id\"\n    ", "start": 1721415264673, "stop": 1721415264713, "uuid": "6e96aabc-45b4-459b-a528-3612779620e0", "historyId": "2092fdbd61fac7323a46276a31f7c15e", "testCaseId": "2092fdbd61fac7323a46276a31f7c15e", "fullName": "tests.users.test_users#test_update_user_with_incorrect_user_id_027", "labels": [{"name": "tag", "value": "users"}, {"name": "parentSuite", "value": "tests.users"}, {"name": "suite", "value": "test_users"}, {"name": "host", "value": "DESKTOP-55MLA3F"}, {"name": "thread", "value": "23412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.users.test_users"}]}